packages/cloud_functions/cloud_functions/lib/src/firebase_functions.dart

```diff
@@ -60,12 +60,43 @@ class FirebaseFunctions extends FirebasePluginPlatform {
   String? _origin;
 
   /// A reference to the Callable HTTPS trigger with the given name.
-  HttpsCallable httpsCallable(String name, {HttpsCallableOptions? options}) {
+  ///
+  /// Should be the name of the Callable function in Firebase
+  /// or the URL of the 2nd gen Callable function in Firebase.
```

Comment:
This comment is incorrect @Lyokone

Reply:
Fixed in https://github.com/firebase/flutterfire/pull/10642